source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/lib/index.ts

```diff
@@ -70,7 +83,16 @@ export class EventbridgeToLambda extends Construct {
       })
     };
 
-    const defaultEventsRuleProps = defaults.DefaultEventsRuleProps([lambdaFunc]);
+    // Check whether an existingBus is provided
+    if (props.existingEventBus) {
+      this.eventBus = props.existingEventBus;
+    } else if (props.eventBusProps) {
+      // eventBusProps is provided, create a new EventBus
+      const _eventBusName = props.eventBusProps.eventBusName || 'CustomEventBus';
+      this.eventBus = new events.EventBus(this, _eventBusName, props.eventBusProps);
+    }
+
+    const defaultEventsRuleProps = defaults.DefaultEventsRuleProps([lambdaFunc], this.eventBus);
     const eventsRuleProps = overrideProps(defaultEventsRuleProps, props.eventRuleProps, true);
 
     this.eventsRule = new events.Rule(this, 'EventsRule', eventsRuleProps);
```

Comment:
since `default` EventBus is always there, CDK construct `events.Rule` doesn't mandate to pass `IEventBus` always as the parameter. If not provided, `default` EventBus is assumed.

Reply:
I missed the last arg on DefaultEventsRuleProps, I see it now.